### DISCUSSION TITLE
Metasploit Cask install failed - download not possible

### DISCUSSION THREAD

**Discussion by 0xBl4ckK1r4** [Homebrew/discussions#4259] [Casks] [answered]
### Output of `brew config`

```shell
$ brew config
HOMEBREW_VERSION: 4.0.3
ORIGIN: https://github.com/Homebrew/brew
HEAD: ba6d87eed9ce31f42fa2b33a4e6067d9cf17f13f
Last commit: 2 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 3e0f7cf00d2e1b9ea58aeee6009c3c98dd3fc3fe
Core tap last commit: 73 minutes ago
Core tap branch: master
Core tap JSON: 22 Feb 08:15 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit westmere
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: true
```

### Output of `brew doctor`

```shell
$ brew doctor                                           
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  python-typing-extensions
  nmap
  python@3.11
  telnet
```

### Description of issue

After updating brew and trying to install updates on the metasploit cask the following error is thrown:

curl: (22) The requested URL returned error: 404

Error: Download failed on Cask 'metasploit' with message: Download failed: https://osx.metasploit.com/metasploit-framework-6.3.3%2B20230213112950-1rapid7-1.x86_64.pkg

Looking at the osx.metasploit.com site this version is no longer there but is written to be the one Homebrew wants to install 
Also the newest version is 6.3.4 and no longer the 6.3.3

![image](https://user-images.githubusercontent.com/110168610/220563653-b3dd58ca-be23-43c1-85b3-9fc7fd0ad39f.png)
![image](https://user-images.githubusercontent.com/110168610/220563922-4bea5b68-2db7-4f46-a2e9-bff5f0cf9875.png)

Is this a problem on my part? 

### Relevant casks

metasploit

**Comment by miccal** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask/pull/141679
